app/ui-message/client/message.js

```diff
@@ -592,6 +592,16 @@ const processSequentials = ({ currentNode, settings, forceDate, showDateSeparato
 	}
 };
 
+export const handleSequential = (message) => {
+	const currentNode = $(`#${ message._id }`);
+	const nextNode = currentNode.next();
+	if (!currentNode.hasClass('sequential')) {
+		if (nextNode !== undefined && nextNode !== null && nextNode.hasClass('sequential')) {
```

Comment:
@ritvikjain99 You can put all the checks in a single if since there is no else.

Reply:
Sure, thanks for suggestions!
Actually with this I aimed readability.